Sources/DocCDocumentation/DocCDocumentation.docc/customizing-your-documentation-appearance.md

```diff
@@ -0,0 +1,244 @@
+text in code voice and another for all other text content. With
+`theme-settings.json`, you can customize the CSS `font-family` lists for these
+however you choose.
+
+**Example**
+
+```json
+{
+  "theme": {
+    "typography": {
+      "html-font": "\"Comic Sans MS\", \"Comic Sans\", cursive",
+      "html-font-mono": "\"Courier New\", Courier, monospace"
+    }
+  }
+}
+```
+
+![A screenshot of a documentation page with custom fonts](theme-screenshot-04)
+
+### Borders
+
+Another aspect of the renderer that you may wish to modify is the way that it
+applies border to certain elements. With `theme-settings.json`, you can change
+the default border radius for all elements and also individually update the
+border styles for asides, buttons, code listings, and tutorial steps.
+
+**Example**
+
+```json
+{
+  "theme": {
+    "aside": {
+      "border-radius": "6px",
+      "border-style": "double",
+      "border-width": "3px"
+    },
+    "border-radius": "0",
+    "button": {
+      "border-radius": "8px",
+      "border-style": "solid",
+      "border-width": "2px"
+    },
+    "code": {
+      "border-radius": "2px",
+      "border-style": "dashed",
+      "border-width": "2px"
+    },
+    "tutorial-step": {
+      "border-radius": "12px"
+    }
+  }
+}
+```
+
+![A screenshot of a tutorial page with custom button
+borders](theme-screenshot-05.png)
+
+![A screenshot of tutorial steps with custom borders](theme-screenshot-06.png)
+
+![A screenshot of a documentation page with custom code listing and aside border
+styles](theme-screenshot-07.png)
+
+### Icons
+
+It is also possible to use `theme-settings.json` to specify custom icons to use
+in place of the default ones provided by DocC-Render. There are a few basic
+steps to follow in order to override an icon:
+
+1. Find the identifier that maps to the icon you want to override.
+2. Add this value as an `id` attribute to the new SVG icon.
+3. Associate a relative or absolute URL to the new SVG icon with the identifier
+   in `theme-settings.json`.
+
+> Note:
+> You can find all the possible identifier values for every valid icon kind in
+> the Open API [specification][1] for `theme-settings.json`.
+
+**Example**
+
+As a simple example, here is how you could update the icon used for articles in
+the sidebar to be a simple box:
+
+1. Find the identifier for article icons: `"article"`
+2. Add the `id="article"` attribute to the new SVG icon: `simple-box.svg`
+    ```svg
+    <svg viewBox="0 0 14 14" xmlns="http://www.w3.org/2000/svg" id="article">
+      <rect x="2" y="2" width="10" height="10" stroke="currentColor" fill="none"></rect>
+    </svg>
+    ```
+3. Configure `theme-settings.json` to map the URL for the new article icon
+    ```json
+    {
+      "theme": {
+        "icons": {
+          "article": "/images/simple-box.svg"
+        }
+      }
+    }
+    ```
+
+![A screenshot of the sidebar using a custom icon for
```

Comment:
Not really sure how this is possible, but in the screenshot it looks like the hero background is bleeding past the sidebar divider.

Reply:
Whoops, I'll look into that. There can be some odd issues that pop up when taking screenshots from the browsers' developer tools.

Reply:
I updated these screenshots with f20fdcdca9025c3be0344ef458a4d2a7121b58da